FEXCore/Source/Interface/Core/OpcodeDispatcher.cpp

```diff
@@ -2285,10 +2310,20 @@ void OpDispatchBuilder::RCLOp(OpcodeArgs) {
   Ref Src = LoadSource(GPRClass, Op, Op->Src[1], Op->Flags, {.AllowUpperGarbage = true});
   const auto OpSize = OpSizeFromSrc(Op);
 
+  auto CalculateZeroResult = [this, Op]() {
```

Comment:
Can we promote this to `OpDispatchBuilder::` space so we don't duplicate it across each rotate instruction?

Reply:
promoted